src/otx/algo/detection/huggingface_model.py

```diff
@@ -72,6 +72,7 @@ def __init__(
         self.load_from = None
 
         super().__init__(
+            model_version=model_name_or_path,
```

Comment:
How about removing `self.model_name` from this class? It seems that class has two duplicated attirbutes.

Reply:
Done 4277eba